internal/lint/subscribe.go

```diff
@@ -12,35 +12,56 @@ import (
 
 // SubscriberInput for the shared test.
 type SubscriberInput struct {
-	// Subject should return a subject to be tested, with given topic/messages seeded.
+	// Subject should be set to a subject to be tested.
+	// It can optionally prepare (pre-create) or seed the topic at this point (the same topic will be used for Seed-ing later).
 	// Caller should handle all the subject teardown/cleanup.
-	// Topics are guaranteed to be prefixed with "bps-unittest-".
 	Subject func(topic string, messages []bps.SubMessage) bps.Subscriber
```

Comment:
I'd rather not pass messages to `Subject` AND to `Seed`, one or the other

Reply:
Addressed lint-related suggestions in https://github.com/bsm/bps/pull/31